packages/dashboard/src/components/side-panel/sub-components/properties-alarms-section.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ExpandableSection } from '@cloudscape-design/components';
+import ExpandableSectionHeader from '../shared/expandableSectionHeader';
+
+const PropertiesAlarmsSection = () => {
+  return (
+    <ExpandableSection
+      headerText={<ExpandableSectionHeader text="Properties & Alarms" onClickButton={() => {}} />}
```

Comment:
This is inconsistently either a `string` or an `ExpandableSectionHeader` in these sub-components?